src/modelAnalysis/coupling/constrainRxnListAboveBound.m

```diff
@@ -0,0 +1,54 @@
+function modelConstrained=constrainRxnListAboveBound(model,rxnList,c,d,csense)
+%
+% Constrains a (weighted) sum of absolute fluxes to be above a lower bound.
+%
+%
+% INPUTS:
+%    model:         model structure
+%    rxnList:       cell array of reaction names
+%
+% OPTONAL INPUTS:
+%    C:             k x n matrix in C*v>=d
+%    d:             n x 1 vector C*v>=d
+%    csense:        k x 1 constraint sense
+%
+% OUTPUT:
+%    modelConstrained:  constrained model
+
+[nMet,nRxn]=size(model.S);
+
+% Identify the indices of these reactions in the new model
+rxnInd = findRxnIDs(model, rxnList);
+
+modelConstrained=model;
+C=sparse(1,nRxn);
+if ~exist('c', 'var')
+    modelConstrained.C(1,rxnInd)=1;
```

Comment:
Could you update this so that subsequent calls to this function update the model adding rows to the Constraint matrix? 
I know, you can provide a C matrix, but for users it would be much more convenient, to just add all their constraints one after another).